app/bin/tools/remove_all_analysis.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:gcloud/db.dart';
+
+import 'package:pub_dartlang_org/analyzer/models.dart';
+import 'package:pub_dartlang_org/frontend/service_utils.dart';
+
+Future main(List<String> arguments) async {
+  await withProdServices(() async {
```

Comment:
Maybe add a comment or something on when we should run this and remove the tool itself..